### DISCUSSION TITLE
When the moviepy  version2.0 publish？

### DISCUSSION THREAD

**Discussion by fangshliu** [Zulko/moviepy#1942] [General]
I watch the project long times, the project is very interesting and useful。

When the moviepy  project version2.0 publish？

**Comment by mgaitan** [comment]
@fangshliu we hope it be soon. Meanwhile you can help us by installing the current development "master" branch with 

```
pip install git+https://github.com/Zulko/moviepy.git
```

Or even better, try the new feature from #1076 and give me feedback.

**Reply by fangshliu** [reply]
Thanks，i will try to using the dev branch .

**Reply by wgong** [reply]
Started using moviepy recently, it is awesome, will help test v2 too
